_studio/mfx_lib/cmrt_cross_platform/include/cmrt_cross_platform.h

```diff
@@ -52,6 +52,9 @@
 #if defined(__clang__)
   #pragma clang diagnostic push
   #pragma clang diagnostic ignored "-Wnon-virtual-dtor"
+
```

Comment:
Again, change in this file, in this file _alone_, should be placed in a separate commit!